### ISSUE TITLE
Profiles menu doesn't fully expand when near the top of the screen

### ISSUE THREAD

**Issue by PhilBaz** [Beep6581/RawTherapee#5699]
Please fix the  'processing profile' popup menu.....

It deploys very oddly when clicked on. First, a menu showing only the word custom shows with an arrow above and below it presented. Then, after scrolling up with the arrow the base menu is presented with flyout arrows for the next level. The button should take you directly here. Also, on clicking the menu should open downward from the button, not up/up and down.

Thanks!

**Comment by Floessie** [comment]
OS? Environment (X11/Wayland)? RT version?

Thanks!

**Comment by PhilBaz** [comment]
ahhh, sorry!!!

Windows 10.... 5.8 stable release.

**Comment by Thanatomanic** [comment]
I agree that this is not ideal, but I'm not sure we can fix this (could be GTK related). It is less annoying when your screen is not maximized, but yeah, who doesn't do that...

**Comment by PhilBaz** [comment]
Ahhh, that's a shame. Thanks for looking at it,
